### PR TITLE
Quick tool to generate lists of codepoints from cmap data.

### PR DESCRIPTION
**Pull request by dougfelt**

### CODE REVIEW

File: nototools/gen_cplist.py [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+  if len(scripts) != len(namepats):
+    if len(namepats) != 1:
+      raise ValueError(
+          'Have %d script%s but %d namepats' %
+          (len(scripts), '' if len(scripts) == 1 else 's', len(namepats)))
+    if '%s' not in namepats[0] and len(scripts) > 1:
+      raise ValueError(
+          'Have multiple scripts but single namepat "%s" has no substitution'
+          % namepats[0])
+  if len(namepats) < len(scripts):
```
Comment (jamesgk) [inline nototools/gen_cplist.py]: We don't need this if statement, right? Just put the following line in the preceding block I think.

File: nototools/gen_cplist.py [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+      '-c', '--cmap_file', help='cmap data file to use (default %s)' %
+      default_cmap, default=default_cmap, metavar='file')
+  parser.add_argument(
+      '-d', '--dest_dir', help='directory for output, (defaults to current '
+      'directory)', metavar='dir', default='.')
+  parser.add_argument(
+      '-s', '--scripts', help='script ids of data to output', nargs='+',
+      metavar='id', required=True)
+  parser.add_argument(
+      '-n', '--namepats', help='name patterns (default "cps_%%s.txt")',
```
Comment (jamesgk) [inline nototools/gen_cplist.py]: Might want to specify what this name pattern is used for here.

File: nototools/gen_cplist.py (hunk 1 above)
Comment (dougfelt) [inline nototools/gen_cplist.py]: Yeah, I completely missed that, thanks.

File: nototools/gen_cplist.py (hunk 2 above)
Comment (dougfelt) [inline nototools/gen_cplist.py]: OK, expanded the help string a bit.

**Pull request merged**